### PR TITLE
Add edvorakjp layout for the Iris keyboard

### PR DESCRIPTION
**Pull request by epaew**

### CODE REVIEW

File: keyboards/iris/keymaps/edvorakjp/keymap.c [hunk 1]
```diff
@@ -0,0 +1,171 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```
Comment (drashna) [inline keyboards/iris/keymaps/edvorakjp/keymap.c]: Heads up, the action_layer and eeconfig includes aren't actually needed.  

No need to change this, but just wanted to let you know.

File: users/edvorakjp/edvorakjp.c [hunk 2]
```diff
@@ -0,0 +1,246 @@
+#include "eeprom.h"
+#include "edvorakjp.h"
+
+bool japanese_mode;
+uint16_t time_on_pressed;
+
+edvorakjp_config_t edvorakjp_config;
+
+uint8_t eeconfig_read_edvorakjp(void) {
```
Comment (drashna) [inline users/edvorakjp/edvorakjp.c]: Very cool on the eeprom stuff!

**Comment by drashna** [comment]
I've flagged a few things, they're all optionally.  But just in case you want to change them.... let me know when this is ready to go.

**Comment by epaew** [comment]
@drashna 
I've fixed where I want to, so I want you to merge this request! Thanks!!

**Comment by drashna** [comment]
Welcome! 
And thank you!

**Pull request merged**